wallet/src/views/wallet/index.jsx

```diff
@@ -14,77 +14,427 @@ import Header from '@Components/Header/header.jsx';
 import SideBar from '@Components/SideBar/index.jsx';
 import Tokens from '@Components/Tokens/index.jsx';
 import { getWalletBalance } from '@Nightfall/services/commitment-storage.js';
+import Lottie from 'lottie-react';
 import { UserContext } from '../../hooks/User/index.jsx';
 
 import './wallet.scss';
 import * as Storage from '../../utils/lib/local-storage';
 import Web3 from '../../common-files/utils/web3';
 import { useAccount } from '../../hooks/Account/index.tsx';
 import useInterval from '../../hooks/useInterval.js';
+import {
+  getIndexedDBObjectRowsFromBackupFile,
+  convertFileToObject,
+  addObjectStoreToIndexedDB,
+} from '../../utils/CommitmentsBackup/import';
+import isCommitmentsCPKDMatchDerivedKeys from '../../utils/CommitmentsBackup/commitmentsVerification.js';
+import successHand from '../../assets/img/modalImages/success-hand.png';
+import checkMarkCross from '../../assets/lottie/check-mark-cross.json';
 
 const supportedTokens = importTokens();
 
 const { DEFAULT_ACCOUNT_NUM } = global.config;
 
 const { ethereum } = global;
 
-/**
-This is a modal to detect if a wallet has been initialized
-*/
-
 function WalletModal(props) {
   const [, , deriveAccounts] = useContext(UserContext);
   const [screenMnemonic, setScreenMnemonic] = useState();
```

Comment:
Hey @luizoamorim, I think it is the first time I see this file, I'm sorry if I missed it, but I have a concern about it since it is storing the mnemonic in attributes/variables, which is a vulnerability. Is there how to get rid of these fields which store them?
This kind of sensitive information should be very very short-lived, I mean, as soon as we use it, we must blank/nullify the variable/attribute so that it doesn't hang on the memory.

Reply:
At this first moment I don't see other way to handle this. Because the user will generate it from the screen and we need to manipulate this info. 
What I did, in the last commit, was clear the variable state as soon that its is used. Anyways, after the modal is closed the variable is lost, because it is a local variable.
Anyways we can open an other issue to investigate a better way to track this. Maybe we can use an env var with some word to encrypt and decrypt this field between it is handled.

Reply:
> 

Yes, I understand that the object will be garbage collected at some moment, but the value of the variables/attributes will still hold in memory until their respective memory addresses are used by another processes.